benchmarks/infra/stage-1/variables.tf

```diff
@@ -36,10 +36,14 @@ variable "gke_location" {
   default     = "us-central1-a"
 }
 
-variable "enable_private_endpoint" {
-  description = "When true, the cluster's private endpoint is used as the cluster endpoint and access through the public endpoint is disabled."
-  type        = bool
-  default     = true
+variable "private_cluster_config" {
```

Comment:
It looks like we were defaulting to a private cluster before. Now, with null default, it changes it to a public cluster. Is that right?

Reply:
No, private remains the default. The default is {}, which I think then gets set to the object with variable enable_private_endpoint = true & master_global_access = true. That seems to be how it works given my two tests of {} vs null + terraform plan.